lib/travis/api/app.rb

```diff
@@ -93,7 +94,7 @@ def initialize
         end
 
         use Travis::Api::App::Cors # if Travis.env == 'development' ???
-        use Raven::Rack if Endpoint.production? && Travis.config.sentry.dsn
+        use Raven::Rack if Travis.env == 'production' || Travis.env == 'staging'
```

Comment:
That's weird, what kind of behaviour do you get when you put `self.class.use_monitoring?` here?